doc/apidoc.rst

```diff
@@ -0,0 +1,45 @@
+=============================
+dbxincluder API documentation
+=============================
+
+Internals
+=========
+
+dbxincluder is divided into two parts.
+
+The xinclude submodule handles the pure XInclude 1.1 specification.
+It was necessary to re-implement it in python as libxml2 does not handle all necessary XInclude 1.1 features itself.
+
+When processing a document, it is first run through the XInclude processor, then the DocBook transclusion processor.
+
+
+dbxincluder.xinclude
+====================
+
+The XInclude process works in two stages.
+
+The first stage handles all xi:include elements by replacing them with either the target subdocument
```

Comment:
Use backticks to make `xi:include` appear in monospace (same for `xi:fallback` and other elements).